doc/tech/release_notes/baldy.md

```diff
@@ -0,0 +1,121 @@
+| `baldy.rm.7` 	| Generic Installer Model 	| Ch09 	|
+| `baldy.rm.8` 	| Guidelines 	| Appnedix-A 	|
+
+*Baldy Release should include at least features up to and including `baldy.rm.4`.
+
+<a name="ra1"></a>
+## Reference Architecture 1 (v2.0)
+
+>WSL Note: Features below should be implemented in order.
+
+| Reference # 	| Feature 	| Notes 	|
+|-------------	|-----------------	|-------	|
+| `baldy.ra1.1` 	| General Cleanup 	| All Chapters 	|
+| `baldy.ra1.2` 	| Clarify OpenStack version policy 	| Ch01, Ch05 	|
+| `baldy.ra1.3` 	| Align Requirement to RM requirements 	| Ch02 	|
+| `baldy.ra1.4` 	| Create a proposal for an exiting Gap 	|  Ch08	|
+| `baldy.ra1.5` 	| Complete High Level Architecture 	| Ch03 	|
+| `baldy.ra1.6*` 	| Complete Interfaces & APIs 	|  Ch05	|
+| `baldy.ra1.7` 	| Complete Components Level Architecture 	| Ch04 	|
+| `baldy.ra1.8` 	| Complete Security Chapter 	|  Ch06	|
+| `baldy.ra1.9` 	| Complete LCM Chapter 	|  Ch07	|
+
+*Baldy Release should include at least features up to and including `baldy.ra1.6`.
+
+
+<a name="ri1"></a>
+## Reference Implementation 1 (v1.0)
+
+<p align="center"><img src="../figures/ri1_baldy.png" alt="scope" title="Document Types" width="100%"/></p>
+<p align="center"><b>Figure 1:</b> RI-1 Baldy Release plan</p>
+
+>WSL Note: Features below should be implemented in order.
+
+| Reference # 	| Feature 	| Notes 	|
+|-------------	|-----------------	|-------	|
+| `baldy.ri1.1` 	| General Cleanups 	| All Chapters 	|
+| `baldy.ri1.2` 	| Complete Overall Requirements 	| Ch02 	|
+| `baldy.ri1.3` 	| Complete Lab Requirements 	| Ch04 	|
+| `baldy.ri1.4` 	| Complete Target State & Metadata 	| Ch03 	|
+| `baldy.ri1.5` 	| Complete Installer Requirements 	| Ch05 	|
+| `baldy.ri1.6` 	| Complete Lab Cookbook (Ops) 	| Ch06 	|
+| `baldy.ri1.6` 	| Restructure & Complete Integration Cookbook	| Ch07 	|
+| `baldy.ri1.7*` 	| Implement Profiles within OPNFV Installers and consume CNTT metadata 	|  **DEV**	|
+
+*Baldy Release should include at least features up to and including `baldy.ri1.7`.
+
+<a name="rc1"></a>
+## Reference Conformance 1 (v1.0-Pre-alpha)
+
+<p align="center"><img src="../figures/rc1_baldy.png" alt="scope" title="Document Types" width="100%"/></p>
+<p align="center"><b>Figure 2:</b> RC-1 Baldy Release plan</p>
+
+>WSL Note: Features below should be implemented in order.
+
+| Reference # 	| Feature 	| Notes 	|
+|-------------	|-----------------	|-------	|
+| `baldy.rc1.1` 	| General Cleanup 	| All Chapters 	|
+| `baldy.rc1.2` 	| Clarify Conformance Categories (NFVI & VNFs)	| Ch01 	|
+| `baldy.rc1.3` 	| Complete NFVI Framework Requirements 	| Ch02 	|
+| `baldy.rc1.4` 	| Categorise NFVI TC Req and Write API Testing TC 	|  Ch03	|
+| `baldy.rc1.5` 	| Create NFVI Mapping & Traceability Matrix and populate it 	| Ch05 	|
+| `baldy.rc1.6` 	| Restructure NFVI Cookbook and Cleanup  	| Ch04 	|
+| `baldy.rc1.7*` 	| NFVI Framework & Test Cases Development   | **DEV** 	|
```

Comment:
There is an RC WS Dev [A: (General) RI-1 passes the RC-1 test suite execution (For sanity and APIs)](https://wiki.lfnetworking.org/display/LN/CNTT+Baldy) work item re: successfully passing RI-1/RC-1 validation suites with Baldy.  Does this work item need to be added to, or does it equate to baldy.rc1.7?

Reply:
@mf4716 you are quite write, added.

Reply:
It should be the opposite - RI should pass RC.
For the time being RC is being verified by itself (see Functest gates).